src-v2/Not/Not.MAUI/Not.MAUI.csproj

```diff
@@ -19,8 +19,8 @@
 	</PropertyGroup>
 
 	<ItemGroup>
-	  <PackageReference Include="Microsoft.Extensions.Localization.Abstractions" Version="7.0.5" />
-	  <PackageReference Include="MudBlazor" Version="6.11.0" />
+	  <PackageReference Include="Microsoft.Extensions.Localization.Abstractions" Version="7.0.16" />
+	  <PackageReference Include="MudBlazor" Version="6.17.0" />
```

Comment:
Why did you update the nuggets? Initially I thought that you had some sort of version mismatch between projects, but I don't think that's the case. 

Generally speaking you should not carelessly update nuggets, even feature;/patches unless you have reason, because this is an easy way to cause bugs. That said it is good to be on top of dependencies, because they often fix performance and security bugs. But this should always be a standalone code-change and should be paired with extensive testing.